### ISSUE TITLE
format of importing 

### ISSUE THREAD

**Issue by Tsang-Kai** [nus-cs2030/2223-s1#42]
to import methods like Math.atan, what is the preferred way of importing?
should we import java.lang.Math and use Math.atan,
or import static,
or import the specific method only?

**Comment by kohlindy** [comment]
import java.lang.Math and use Math.atan2

**Comment by wilsonwid** [comment]
I don't think we need to write in `import java.lang.*` though, as the default Java library is already imported.

**Comment by danqiye1** [comment]
I guess I can state one preference for importing here which is to not do `import java.lang.*`. The use of wildcard `*` is ambiguous to readers of your code on what you actually imported and intend to use.

>or import the specific method only?

I don't think you can import a method only in java, since everything is encapsulated in some class.

>import static

You probably won't encounter a need for this.
